CMakeLists.txt

```diff
@@ -36,6 +36,11 @@ if(CMAKE_VERSION VERSION_GREATER_EQUAL "3.24.0")
   cmake_policy(SET CMP0077 NEW)
 endif()
 
+# Avoid warning of Calling FetchContent_Populate(GSL) is deprecated
```

Comment:
Thanks for the fix. How did you find it? CMake 3.30 is not release yet.

Reply:
I tried several versions of cmake local installer and noticed this coming change.